axelnguyen0701/book-exchange
# Close listing axel

## Changes
- Close listing by instant price
- See owned listings
## Steps to test
- You will need to import at least 2 accounts from hardhat
- On one account, create a listing. Then on the other account, buy that listing
- On the buyer account, go to "My Books" on the navbar to see your own listings
